graphprot/DataSet.py

```diff
@@ -199,7 +199,11 @@ def load_one_graph(self, fname, mol):
         #print('Load mol :', mol)
 
         f5 = h5py.File(fname, 'r')
```

Comment:
a usual way of writing is is 
```python
with h5py.File(fname,'r') as f5:
     grp = f5[mol]
```
so that you are sure that the file is closed when the scope of the `with` statement is finshed